modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/view.jsp

```diff
@@ -222,26 +224,55 @@ long cpDefinitionId = cpCatalogEntry.getCPDefinitionId();
 List<CPDefinitionSpecificationOptionValue> cpDefinitionSpecificationOptionValues = cpContentHelper.getCPDefinitionSpecificationOptionValues(cpDefinitionId);
 List<CPMedia> cpMedias = cpContentHelper.getCPMedias(cpDefinitionId, themeDisplay);
 List<CPOptionCategory> cpOptionCategories = cpContentHelper.getCPOptionCategories(company.getCompanyId());
+String description = cpCatalogEntry.getDescription();
+boolean hasCPDefinitionSpecificationOptionValues = cpContentHelper.hasCPDefinitionSpecificationOptionValues(cpDefinitionId);
+String navCPMediaId = "navUnderlineCPMedia";
+String navDescriptionId = "navUnderlineDescription";
+String navSpecificationId = "navUnderlineSpecifications";
 %>
 
-<c:if test="<%= cpContentHelper.hasCPDefinitionSpecificationOptionValues(cpDefinitionId) %>">
-	<commerce-ui:panel
-		elementClasses="flex-column mb-3"
-		title='<%= LanguageUtil.get(resourceBundle, "specifications") %>'
-	>
+<div>
+	<react:component
+		module="product_detail/render/js/Tabs"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"hasCPDefinitionSpecificationOptionValues", hasCPDefinitionSpecificationOptionValues
+			).put(
+				"hasCPMedia", !cpMedias.isEmpty()
+			).put(
+				"hasDescription", (description != null) && (description.length() > 0)
+			).put(
+				"navCPMediaId", navCPMediaId
+			).put(
+				"navDescriptionId", navDescriptionId
+			).put(
+				"navSpecificationId", navSpecificationId
+			).build()
+%>'
+	/>
+</div>
+
+<div class="tab-content" id="tab-content" style="background: #ffffff;">
```

Comment:
Use another ID and prefix it with portlet namespace, use camel case. Write CSS instead of using the style property.